src/pages/Host/DragDrop/Dropzone.js

```diff
@@ -0,0 +1,82 @@
+import React, { useContext, useRef, useState } from 'react';
+import HostStore, { HostContext } from '../User';
+import styled from 'styled-components/macro';
+import variables from '../../../styles/variables';
+
+function DropzoneComponent() {
+  const context = useContext(HostContext);
+  const { imgs, setImgs } = context;
+  const [fileImage, setFileImage] = useState('');
+
+  const onImageChange = e => {
+    const imgFile = e.target.files;
+    setFileImage(URL.createObjectURL(e.target.files[0]));
+
+    if (imgs.length < 5) {
+      setImgs([...imgs, imgFile]);
+    }
+  };
+
+  const inputref = useRef(null);
+  const UseRef = e => {
+    inputref.current.click();
+  };
```

Comment:
????! 함수명이 조금 독특합니다.

```suggestion
  const onClickMessageBox = e => {
    inputref.current.click();
  };
```
이렇게 가는건 어떄요?
useRef는 react에서 import해왔기 떄문에 UseRef로 함수명을 지으신 것 같은데, 함수가 하는 행동은 `MessageBox를 클릭하였을 때` 인 것 같습니다.